### ISSUE TITLE
Microphone icon?

### ISSUE THREAD

**Issue by ChristopherMori** [dgonano/MMM-AlexaPi#3]
Added the module to my config file but when I run Magic Mirror all I see is a Microphone crossed out. The status doesn't seem to change at all.

**Comment by dgonano** [comment]
Have you installed AlexaPi? This module is the Mirror side of the connection between the two programs

**Comment by dgonano** [comment]
Also if you can provide configuration files or some logs or something so I can try and see the issue.

**Comment by ChristopherMori** [comment]
Yup Alexa Pi is installed and functioning. What type of logs do you need/ want?

**Comment by pleasurous** [comment]
Hello dgonano, i am facing the same problem here. already cloned git and add module in array config but it seems not working at all. there are icon cross mic icon however i have AlexaPi installed and running well on my raspi3. do i need to add something somewhere on mmm-alexapi module setting?

**Comment by ChristopherMori** [comment]
Same configuration as pleasurous

**Comment by dgonano** [comment]
@ChristopherMori sorry for the late reply! 
For both of you, everything should work with the default settings assuming your using defaults for other stuff. 
I don't actually have my system with my so I can't test latest versions unfortunately. 
If you need help please post both config files or some logs. 
The mirror config should just have the module with a position and the alexapi config should just be set to `magicmirror`. Is alexapi set to `magicmirror` or `raspberrypi`?

**Comment by pleasurous** [comment]
@dgonano thank you for your response..
i am now understand how to make this done. 
because, when the first install ```AlexaPi``` it give me hint ```device[raspberrypi]:``` and whe i didn't fill it as hint it caused cancel installation so as default it set to ```raspberrypi```.


```
sudo nano /etc/opt/AlexaPi/config.yaml
```

then edit 
```
platform:
  # Name of your platform, e.g. raspberrypi, orangepi, desktop
  device: "raspberrypi"
```
to
```
platform:
  # Name of your platform, e.g. raspberrypi, orangepi, desktop
  device: "magicmirror"
```
 
@ChristopherMori you could try those step.

**Comment by ChristopherMori** [comment]
I left it as raspberrypi during the original Alexa Pi setup.

**Comment by anthony6608** [comment]
> I left it as raspberrypi during the original Alexa Pi setup.

@pleasurous Thanks, I have the same problem and trying your solution in hopes of fixing it.

@ChristopherMori A thank you to @pleasurous would be nice.... he actually went out of his way to reply back to you to make sure you got it working. Smh.

**Comment by anthony6608** [comment]
> @dgonano thank you for your response..
> i am now understand how to make this done.
> because, when the first install `AlexaPi` it give me hint `device[raspberrypi]:` and whe i didn't fill it as hint it caused cancel installation so as default it set to `raspberrypi`.
> 
> ```
> sudo nano /etc/opt/AlexaPi/config.yaml
> ```
> then edit
> 
> ```
> platform:
>   # Name of your platform, e.g. raspberrypi, orangepi, desktop
>   device: "raspberrypi"
> ```
> to
> 
> ```
> platform:
>   # Name of your platform, e.g. raspberrypi, orangepi, desktop
>   device: "magicmirror"
> ```
> @ChristopherMori you could try those step.

Thanks again! I followed your instructions, rebooted and everything is working fine now. Much appreciated!
